simulariumio/dep/pyMCDS.py

```diff
@@ -0,0 +1,548 @@
+import xml.etree.ElementTree as ET
+import numpy as np
+import pandas as pd
+import scipy.io as sio
+import sys
+import warnings
+from pathlib import Path
+
+"""
+Source: https://github.com/PhysiCell-Tools/python-loader/blob/master/pyMCDS.py
```

Comment:
To properly attribute this you might want to get the actual commit version url instead of the "master" one, since it might change over time.